packages/safe-tools-client/app/components/setup-safe-modal/index.css

```diff
@@ -0,0 +1,34 @@
+.setup-safe-modal__section {
+  padding: 50px 60px 50px 50px;
```

Comment:
(NO CHANGE REQUIRED) This implementation (which I think is correct) brings up an interesting point re: Boxel's spacing units that I've been thinking about for a while. Not for action in this PR, more generally about the length primitives we're using and how they should interact with user font size preferences.

We use `rem`-based spacing with `--boxel-sp-` variables since this helps things to scale evenly with user preferences for font size, but this is [not always desirable](https://medium.com/@sascha.wolff/dont-use-rem-em-for-paddings-margins-and-more-94e19026b000). I think it's relevant when the spacing communicates text/other grouping, but for things like padding it just reduces the available amount of space for content (that is already taking up more space because of increased font size).